hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -22,6 +22,67 @@
       milestone: V1
       tests: ["i2c_sanity"]
     }
+    {
+      name: error
+      desc: '''
+            Program incorrect configuration into DUT register.
+
+            Stimulus:
+              - Program negative value into timing registers.
+              - Program incorrect format flags into fdata register.
+
+            Checking:
+              - Check involved interrupts/alerts.
+              - Got timeout in test (scoreboard might not get any transaction for DUT).
+            '''
+      milestone: V2
+      tests: ["i2c_error"]
+    }
+    {
+      name: reset_recovery
+      desc: '''
+            Pull reset at random times, make sure DUT recover/resets correctly and
+            there is no residual data left in the registers.
+
+            Stimulus:
+              - Randomly reset DUT.
+              - Restart send/receive transaction.
+
+            Checking:
+              - Check if registers are properly reset.
+            '''
+      milestone: V2
+      tests: ["i2c_reset_recovery"]
+    }
+    {
+      name: perf
+      desc: '''
+            Send/receive transactions at max bandwidth
+
+            Stimulus:
+              - Reduce access latency for fmt_fifo and rx_fifo.
+              - Issue long read/write transaction (and read rx_fifo as soon as read data valid).
+
+            Checking:
+              - None
```

Comment:
Can you add a little more justification why no checking is required?

Reply:
Missing point, fixed in amendment